src/Component/Carousel.jsx

```diff
@@ -0,0 +1,55 @@
+import './Carousel.css';
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css"; 
+import "slick-carousel/slick/slick-theme.css";
+import data from "../assets/images.json";
+import SVG1 from "../assets/Vector-1.svg";
+
+
+export const CarouselComponent = () => {
+    const settings = {
+        centerMode: true, // This enables centered view with partial prev/next slides. Use with odd numbered slidesToShow counts.
+        infinite: true,
+        centerPadding: "0px", // You might need to adjust this padding to get the desired effect.
+        slidesToShow: 1, // This would show 3 slides at a time. The one in the center will be the main focused slide.
```

Comment:
slidesToShow is set to show 1 wich means it will show 1 slide at time, but if you set it to 3 you will see 3 slides with the one in the middle as the focus point just as the comment explains;)